src/api-client/Qbert.js

```
@@ -337,8 +337,12 @@ class Qbert {
 
   /* Managed Apps */
   getPrometheusInstances = async (clusterUuid) => {
-    const response = await this.client.basicGet(`${await this.baseUrl()}/clusters/${clusterUuid}/k8sapi/apis/monitoring.coreos.com/v1/prometheuses`)
-    return normalizeClusterizedResponse(clusterUuid, response)
+    try {
```

I don't think this is the best place to put a `try` `catch` statement. 
If we do this here we should repeat the same in all the other methods of the class. 
Moreover we already have error handling in `createContextLoader` and `createContextUpdater` helpers, which are used everywhere to load and update server data.

---

@xantorres I am not a UI developer, but in my testing, I found that putting try catch here does load prometheus instances. I have an env where one cluster does not have prometheus api extensions while other does. Perhaps you can take a look at the path and tell me where is the try catch being missed?

---

@xantorres The error handling didn't seem to catch this one.  In the interest of getting this feature working I think we can leave the `try` / `catch` here for now and refactor it out later when we can verify that we handle it properly higher up.  The thing it does here is that it returns an empty array in the case of errors which is the desired result in this specific case.  I'm not sure using that same logic higher up would be appropriate at a more generic level.  We would probably want to surface the errors and not mask them out as we are in this case.